IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcLinearZone/DocEntity.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcLinearZone" UniqueId="6b26ca28-e2b8-4750-b7ae-ed25437db4ef" BaseDefinition="IfcLinearElement" EntityFlags="32" />
```

Comment:
Do we make it abstract if we are adding children?

Reply:
Depends. Right now we don't have a comprehensive list of children.

Reply:
Made it abstract since it is a non-leaf entity. See [f5c1884](https://github.com/bSI-InfraRoom/IFC-Specification/pull/455/commits/f5c18846ce333b199543fd0ce1262fc36c73c33a)